docs/setup_remotessh.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: spec
+title: Setup VS Code CAEN Access
+---
+
+VS Code CAEN Access
+==========
+{: .primer-spec-toc-ignore }
+
+This tutorial will show you how to set up VS Code access into CAEN Linux via SSH. It allows you to remotely edit and execute files as if you were using VS Code on the CAEN computers themselves. This may come in handy for those who wish to develop and test their projects directly on CAEN.
+
+
+## Prerequisites
+If you haven't already, follow the [CAEN Linux Tutorial](setup_caen.html) so your CAEN environment is ready for VS Code to access. If you need a refresher on VS code, read through the [tutorial](setup_vscode.html).
+
+
+<div class="primer-spec-callout warning" markdown="1">
+**Pitfall:** If you are off campus, make sure you have connected to the [UM VPN](https://its.umich.edu/enterprise/wifi-networks/vpn/getting-started) before continuing.
+</div>
+
+## Setup Remote SSH Extension
+
+### Edit SSH Config
+First, we need to specify CAEN as a host in our SSH config file so VS Code knows where to connect. You can find the file at `~/.ssh/config`.
+
+Add the following lines to the SSH config file, replacing `<uniqname>` with your own uniqname.
+```
+Host caen #(Can be named anything you like)
+  HostName login.engin.umich.edu
+  User <uniqname>
+```
+
+Verify the contents of your `~/.ssh/config` file.
+```console
+$ cat ~/.ssh/config
+# SSH multiplexing (from CAEN Linux Tutorial)
```

Comment:
Is SSH multiplexing a prerequisite for this tutorial (even if not, it might be useful to specify this).

Reply:
Added. I linked the entire CAEN tutorial under the prerequisites section and this tutorial assumes that you've already completed it. But it's true that students might forget about the multiplexing step.